src/styles/app.scss

```diff
@@ -4,11 +4,6 @@ html, body, #root {
   width: 100%;
 }
 
-#root {
-  overflow-x: hidden;
-  overflow-y: scroll;
-}
```

Comment:
Just want  to make sure, we are still able to scroll on small screen heights? Or when the user has zoomed in?

Reply:
@bent0b0x we are using the default scroll.
I don't get it the point